lib/DDG/Spice/Seasons.pm

```diff
@@ -60,6 +60,10 @@ handle query_lc => sub {
         $country = lc $loc->country_code;
     }
 
+    # Get hemisphere
+    my $hemisphere = ($loc->latitude > 0) ? 'north' : 'south';
```

Comment:
doh! You're right. I wouldn't want to remove the location triggering since it's already there. I'll poke at it some more.